components/form/formitems/template.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Template
+page_title: FormItem - Template
+description: Template for the FormItem.
+slug: form-formitems-template
+tags: telerik,blazor,form,edit,formitems,template
+published: True
+position: 5
+---
+
+# FormItem Template
+
+You can provide your own custom editors instead of the [default editors the form can generate]({%slug form-overview%}#automatic-generation-of-fields). To do that, use the the `Template` of the [FormItem]({%slug form-formitems%}).
+
+When the Template is used, the built-in validation messages from the Form will not be rendered. Instead you can use the [Telerik Validation tools](INCLUDE SLUG HERE) to provide validation messages, or any other suitable tool.
+
+## Use the Template to provide custom editors
+
+````CSHMTL
+@* Use the Template to change the default editors *@
+
+@using System.ComponentModel.DataAnnotations
+
+<TelerikForm EditContext="@EditContext">
+    <FormValidation>
+        <DataAnnotationsValidator></DataAnnotationsValidator>
```

Comment:
Consider adding a validation summary here so the validation is visibly working - there need to be messages somewhere

Reply:
I agree, added it.